src/terminal/parser/tracing.cpp

```diff
@@ -7,11 +7,8 @@
 using namespace Microsoft::Console::VirtualTerminal;
 
 #pragma warning(push)
-#pragma warning(disable : 26494) // _Tlgdata uninitialized from TraceLoggingWrite
-#pragma warning(disable : 26477) // Use nullptr instead of NULL or 0 from TraceLoggingWrite
-#pragma warning(disable : 26485) // _Tlgdata, no array to pointer decay from TraceLoggingWrite
-#pragma warning(disable : 26446) // Prefer gsl::at over unchecked subscript from TraceLoggingLevel
-#pragma warning(disable : 26482) // Only index to arrays with constant expressions from TraceLoggingLevel

```

Comment:
wow! did you test this manually? I love seeing things like this disappear.

Reply:
If I see code like that I usually assume that they're only temporary measures.
Just like I usually do it, I also started deleting this entire block first and then readded only those that were necessary to get it to compile again. As it turns out only a single of those preexisting pragmas was needed after all.